apps/backend/src/lambdas/utils/parseGarms.ts

```diff
@@ -17,60 +17,66 @@ export const parseGarms = async (
   source_file_name: string,
   paymentMethods: PaymentMethodEntity[]
 ): Promise<TransactionEntity[]> => {
-  return garmsJson.map(
-    ({
-      sales_transaction_id,
-      sales_transaction_date,
-      fiscal_close_date,
-      payment_total,
-      payments,
-      source,
-      misc,
-      distributions,
-      revAccounts
-    }: SBCGarmsJson) =>
-      new TransactionEntity({
-        source_id: Ministries.SBC,
-        transaction_id: sales_transaction_id,
-        transaction_date: sales_transaction_date // TODO: just use slice here
-          .split('')
-          .splice(0, 10)
-          .join(''),
-        transaction_time: sales_transaction_date
-          .slice(11, 19)
-          .replaceAll('.', ':'),
-        location_id: parseInt(source.location_id),
-        total_transaction_amount: payment_total,
+  return garmsJson
+    .map((itm) => ({
+      ...itm,
+      payments: itm.payments.filter((payment) => payment.amount !== 0)
```

Comment:
did we find out why certain amounts are 0 in the first place? 

what is the reasoning behind removal?